src/reset/HtmlNav/HtmlNav.tsx

```diff
@@ -6,6 +6,7 @@ import { asPropType } from '../../utils/typescript';
 export interface HtmlNavProps
   extends Omit<HTMLProps<HTMLElement>, 'ref' | 'as'> {
   as?: asPropType;
+  forwardedRef?: React.RefObject<HTMLDivElement>;
```

Comment:
I think the type should be `<HTMLElement>`.  If I have understood correctly nav doesn't have its own type, so `<HTMLElement>` would be the proper one.
And if there is no specific reason for it to be `RefObject` the whole thing should be `React.Ref<HTMLElement>`

Reply:
Fixed this!